.circleci/config.yml

```diff
@@ -41,6 +41,12 @@ jobs:
             python3.6 test_utils/scripts/get_target_packages.py > ~/target_packages
             cat ~/target_packages
       - run:
+          name: Run tests - google.api.core
+          command: |
+            if [[ -n $(grep api_core ~/target_packages) ]]; then
+              nox -f api_core/nox.py
+            fi
```

Comment:
Thanks for reminding me to file https://github.com/GoogleCloudPlatform/google-cloud-python/issues/4211